src/ontology/mondo-edit.obo

```diff
@@ -350706,7 +350712,7 @@ xref: ICD10CM:Q24.5 {source="Orphanet:1081/e", source="Orphanet:1081"}
 xref: icd11.foundation:902783759 {source="MONDO:equivalentTo", source="Orphanet:1081", source="https://orcid.org/0000-0002-4142-7153"}
 xref: MedDRA:10061060 {source="Orphanet:1081/e", source="Orphanet:1081"}
 xref: MEDGEN:1612789 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-xref: Orphanet:1081 {source="MONDO:equivalentTo"}
+xref: Orphanet:1081 {source="MONDO:equivalentTo", source="MONDO:preferredExternal"}
```

Comment:
@sabrinatoro I thought adding `source="MONDO:preferredExternal"` resolved the proxy merge. What am I forgetting 🤔

Reply:
To summarize the chat, I will remove the new xref to Orphanet:1081 on MONDO:0001389 and also remove the MONDO:preferredExternal I added on MONDO:0001390. I will also add a new ticket to merge MONDO:0001389 with MONDO:0001390.

Reply:
New ticket [added](https://github.com/monarch-initiative/mondo/issues/8360).

Reply:
And those changes mentioned above were done.